tools/lkl/cptofs.c

```diff
@@ -493,6 +493,9 @@ int copy_one(const char *src, const char *mpoint, const char *dst)
 		snprintf(dst_path, sizeof(dst_path),  "%s", dst);
 	}
 
+	if (strcmp(src, "/") == 0)
```

Comment:
Is it worth handling `"//"` and other logically equivalent paths here? I don't expect to use them directly, but they can occasionally arise when someone uses programmatically generated paths.